### ISSUE TITLE
Buttons

### ISSUE THREAD

**Issue by NathanWalker** [NativeScript/theme#10]

**Comment by TheOriginalJosh** [comment]
After more testing tonight I've determined that simple multiple classes works for example

```
.btn{
    color:#2B33FF;
}
.btn-primary{
    color:#fff;
}
```

`.btn-primary` will override `.btn`s color value as expected. however, a more complicated styles such as :

```
.btn{
    color:#2B33FF;
    text-transform:capitalize;
    font-size:18
}
.btn-primary{
    color:#fff;
    background-color:#2B33FF;
    margin:8 16 8 16;
    border-width:1;
    border-color:#2B33FF;
    padding:10 10 10 10
}
```
Doesn't work in iOS but does in Android. Instead you will get a mixture of the two and you get:

<img width="359" alt="screen shot 2016-08-11 at 7 59 50 pm" src="https://cloud.githubusercontent.com/assets/1486275/17640733/87cc3586-60d4-11e6-8983-7c7b0c8b8a1e.png">

after more testing and refactoring I've found that apparently `text-transform:capitalize;` appears to be the cause of the white color not overriding the blue color set in the base `.btn` style.  without that in either selector the result is:

<img width="351" alt="screen shot 2016-08-12 at 9 36 07 pm" src="https://cloud.githubusercontent.com/assets/1486275/17640748/eb67c470-60d4-11e6-8549-e4b484eaaf2b.png">
